internal/apiserver/route_get_net_node.go

```diff
@@ -35,7 +35,7 @@ var getNetworkNode = &oapispec.Route{
 	FilterFactory:   nil,
 	Description:     i18n.MsgTBD,
 	JSONInputValue:  nil,
-	JSONOutputValue: func() interface{} { return &fftypes.Node{} },
+	JSONOutputValue: func() interface{} { return &fftypes.Identity{} },
```

Comment:
Note that `Identity` doesn't have exactly the same field set as the old `fftypes.Organization` or `fftypes.Node` objects. However, they are really close and it felt wrong to arbitrarily do mapping.

So we need to describe the difference in the JSON structures (`did` added, `identity` removed, `dx.endpoint` now just `profile`) in the migration notes for the next release.

Reply:
I agree no need to do extra data mapping, but this will be a breaking change for some apps built against the current version of FireFly.

Are we certain we want to keep the old `/nodes` and `/organizations` routes at all, or are the redundant with the new `/identities` routes?